org.gnome.Hitori.json

```diff
@@ -16,11 +16,6 @@
       "--talk-name=ca.desrt.dconf",
       "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
     ],
-  "build-options" :
-    {
-      "cflags": "-O2 -g",
-      "cxxflags": "-O2 -g"
-    },
```

Comment:
The commit message doesn’t mention why the `build-options` have been removed.

Reply:
`flatpak-builder` parses this file which is now in the runtime: `flatpak run --command=cat org.gnome.Sdk//3.30 /etc/flatpak-builder/defaults.json`

Reply:
Nice, thanks for the explanation. This looks good to merge once that’s mentioned in the commit message.